sdk/core/azure-core/tests/testserver_tests/coretestserver/coretestserver/test_routes/streams.py

```diff
@@ -48,7 +59,11 @@ def string():
     return Response(
         streaming_test(), status=200, mimetype="text/plain"
     )
-    
+
+@streams_api.route('/compressed_no_header', methods=['GET'])
+def compressed_no_header():
+    return Response(stream_compressed_no_header(), status=300, headers={"Content-Encoding": "gzip"})
```

Comment:
@marcusc2 On re-review of this, I noticed that this test is for compressed file with _no_ header so we shouldn't send the `Content-Encoding` header here. We do pass the arg `decompress=False` in the actual test so it's not actually looking at this header, but we should remove it so that it matches the intended test case scenario.